src/db/query_methods.cr

```diff
@@ -124,6 +139,23 @@ module DB
       end
     end
 
+    # Executes a *query* that expects a single row and returns it
+    # as a named tuple of the given *types*.
+    #
+    # Returns `nil` if there were no rows.
+    #
+    # Raises `DB::Error` if there were more than one row.
+    #
+    # ```
+    # result = db.query_one? "select name, age from contacts where id = ?", 1, as: {name: String, age: Int32}
+    # typeof(result) # => Tuple(String, Int32) | Nil
+    # ```
+    def query_one?(query, *args, as types : Tuple)
```

Comment:
This definition duplicates the definition directly below it. Perhaps you meant this to be a `NamedTpule` overload? Either way it should be deleted.

Reply:
I noticed that duplicate, but I did not see it in my commit

![https://up.nephos.xyz/17122123/f8a803a4/screenshot_20171221_235536.png](https://up.nephos.xyz/17122123/f8a803a4/screenshot_20171221_235536.png
)

Reply:
Yes I wrote it 6 months ago. It was meant to be the NamedTuple overload, and has been lost with the time :D